docs/src/examples/components/Form/Usage/FormExampleCheckbox.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+import { Form, Button, Checkbox, Provider } from '@stardust-ui/react'
+
+const fields = [
+  {
+    label: 'Email',
+    name: 'email',
+    id: 'email-inline-shorthand',
+    key: 'email',
+    required: true,
+    inline: true,
+  },
+  {
+    control: {
+      as: Checkbox,
+      labelPosition: 'start',
+      label: 'Subscribe to newsletter',
+    },
+    key: 'newsletter',
+    id: 'newsletter-inline-shorthand',
+  },
+  {
+    control: {
+      as: Button,
+      content: 'Submit',
+    },
+    key: 'submit',
+  },
+]
+
+const theme = {
+  componentStyles: {
+    Checkbox: {
+      root: {
+        padding: 0,
+      },
+    },
+  },
+}
```

Comment:
Let's remove this one, we definitely should not show this in examples

Reply:
Without this, however, a padding breaks the visual structure of the form.
![image](https://user-images.githubusercontent.com/10930160/63605218-ca07ff00-c5cd-11e9-978d-be5afda16af7.png)

Reply:
Let design team to care about it, we should tune theme specific things in theme agnostic examples

Reply:
Did you mean `shouldn't`? 😄

Reply:
fixed